Sources/SwiftGen/Config/Config+Run.swift

```diff
@@ -0,0 +1,98 @@
+//
+// SwiftGen
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import AppKit
+import PathKit
+import StencilSwiftKit
+import SwiftGenKit
+
+extension Config {
+  func runActions(verbose: Bool, logger: (LogLevel, String) -> Void = logMessage) throws {
+    let errors = commands.parallelCompactMap { cmd, entry -> Swift.Error? in
+      do {
+        try run(cmd: cmd, entry: entry, verbose: verbose, logger: logger)
+        return nil
+      } catch {
+        return error
+      }
+    }
+
+    if errors.count == 1 {
+      throw errors[0]
+    } else if errors.count > 1 {
+      throw Error.multipleErrors(errors)
+    }
+  }
+
+  private func run(
+    cmd: ParserCLI,
```

Comment:
nit: s/cmd:/parserCommand:/ ? 🤔

Reply:
Dunno 😅 The terminology keeps being difficult:
- SGK has parsers, that's clear
- From CLI we invoke commands
- ParserCLI are an abstraction of a command to invoke a parser
- ParserCLI.command() converts that abstraction to an actual command
- Probably some other stuff I'm missing.

Reply:
Yeah I agree. Naming and terminology is hard.

I want us to be consistent and use the same term everywhere for the same thing to avoid confusion in code and docs, but ParserCLI is indeed not really a parser and more like a command telling what parser to run but also with which in/out parameters, so `command` terminology is probably right here…